src/ng/directive/input.js

```diff
@@ -2142,7 +2142,7 @@ var ngValueDirective = function() {
    *  makes it possible to use ngValue as a sort of one-way bind.
    */
   function updateElementValue(element, attr, value) {
-    element.prop('value', value);
+    element.prop('value', angular.isUndefined(value) ? null : value);
```

Comment:
I am 99% sure it won't make a difference in this particular case, but let's stick to `.prop()` (because...it doesn't make a difference).

BTW, I forgot to mention, you don't need the `angular.` prefix; just `is[Un]Defined(...)`.